File: fix_quotes.js
te_path>
d:/1.Programing/pemograman web/KBT/PortofolioMaker/fix_quotes.js
</absolute_path>
<parameter name="content">
const fs = require('fs');
const path = 'resources/views/index.blade.php';

let content = fs.readFileSync(path, 'utf8');
content = content.replace(/\\'/g, "'");
fs.writeFileSync(path, content);
console.log('Fixed!');

File: fix_quotes.py
# Read the file
with open('resources/views/index.blade.php', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix the escaped quotes: {{ asset(\' should be {{ asset('
content = content.replace("\\'", "'")

# Write the file back
with open('resources/views/index.blade.php', 'w', encoding='utf-8') as f:
    f.write(content)

print('Fixed!')
